docs/Overview.md

```diff
@@ -66,7 +66,7 @@ ReactDOM.render(
 );
 ```
 
-Since the release of React 16.8, use can use [Hooks](https://reactjs.org/docs/hooks-intro.html) as a way to work with `EditorState` without using a class.
+Since the release of React 16.8, we can use [Hooks](https://reactjs.org/docs/hooks-intro.html) as a way to work with `EditorState` without using a class.
```

Comment:
How about "you" instead of "we"?

Reply:
thank you for the response Robbert, `you` is a good option too, you can definitely open a PR and see who gets picked 😉 , to me `we` sounds better personally.

Reply:
Looks like this was already addressed in #2089.

Reply:
Hi @Haobo-Zhao @robbertbrak, thanks for proposing a fix for this typo! I prioritized the earlier PR as I think it's more fair to accept a fix from a contributor who spotted and addressed it first.